MobileCenterDistribute/MobileCenterDistribute/MSDistribute.m

```diff
@@ -530,4 +525,46 @@ - (void)openUrl:(NSURL *)url {
   }
 }
 
+- (NSString *)packageHash {
+
+  /*
+   * BuildUUID is different on every build with code changes.
```

Comment:
This comment is not totally true anymore since the key prefix is now the full hash.